<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>CMB@CU - Computational Molecular Biomedicine at Chulalongkorn University</title>

    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/cmb.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="js/bootstrap.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
    
  </head>
  <body>
    <!-- TOP NAVIGATION BAR -->
	<div ng-app>
      <div ng-include="'m.navbar.html'"></div>
    </div>
	
	<div class="research-banner-bg">
	  <div class="container">
	     <div class="row">
	       <div class="col-md-12" align="right">
	         <h3><font color="grey">Expanding the horizon of computational biology research</font></h3>
	       </div>
	     </div>
	  </div>
	</div>
	
	<!-- RESEARCH -->
	<div class="container-fluid">
	  <div class="row">
	    <div class="col-md-12" align="left">
	      <!-- <h2>Research</h2> -->
	      <hr />
	      <h4>The Computational Molecular Biomedicine group is pursuing 4 major research topics as well as collaborative projects 
	      in diverse biomedical research areas. Most projects combine broad knowledge of molecular biology with state-of-the-art mathematical 
	      modeling and machine learning techniques.</h4>
	    </div>
	  </div>
	  
	  <div class="row">
	  	<div class="col-md-12">
	  	  <hr />
	      <h3><strong>Project 1:</strong> Proteomics data analysis method and software development</h3>
	    </div>
	    <div class="col-md-12">
          <p>The goal of this project is to develop high-performance, high-throughput analysis methods and software tools for proteomics datasets. 
          We are particularly interested in applying machine learning techniques on both CPU and GPU architectures to improve peptide identification 
          and quantification from mass spectrometry data. Example application areas include identification of (i) crosslinked peptides whose mass 
          spectra are generally 3-4 times more complex than those found in typical proteomics studies and (ii) novel peptide sequences and 
          modifications that currently do not exist in reference databases.</p>
          
          
          <!-- 
          <p>Parts of this project will be performed in collaboration with the <a href="http://sysbio.chula.ac.th/home/" target="_blank">Systems Biology Center</a> 
          at Chulalongkorn University (CUSB), whose characterizations of novel proteomes will greatly benefit from improved <i>de novo</i> identification 
          of unknown peptides and proteins, and <a href="https://www.wistar.org/lab/david-w-speicher-phd" target="_blank">Dr. David W. Speicher</a>’s 
          proteomics research laboratory at the Wistar Institute in Philadelphia, PA USA, where zero-length crosslinking is being routinely utilized 
          to probe the structure of red cell membrane proteins and other complex systems.</p> -->
        </div>
      </div>
      
      <div class="row">
	    <div class="col-md-12" align="left">
	      <hr />
	      <h3><strong>Project 2:</strong> Integrative analysis of high-throughput biomedical datasets</h3>
	    </div>
	  </div>
	  <div class="row">
        <div class="col-md-12">
          <p>For this project, our goal is to identify biological bases behind diseases or other complex biological systems through computational analyses of 
          high-throughput biomedical datasets. This can be either hypothesis-driven, where publicly available datasets are used to support/disprove a hypothesis, 
          or data-driven, where emergence of new datasets or technology dictates research direction. Therefore, topics of interest will usually be chosen based 
          on the availability of public databases or via collaborations with experimental researchers from inside and outside of the university.</p>
        </div>
      </div>
	  
	  <div class="row">
	    <div class="col-md-12" align="left">
	      <hr />
	      <h3><strong>Project 3:</strong> Mathematical modeling and simulation of chromatin folding process</h3>
	    </div>
	  </div>
	  <div class="row">
        <div class="col-md-12">
          <p>Genomes are usually thought of as 1-dimensional sequences of genetic codes. However, they are located on chromosomes 
          which fold in 3-dimensional space to form loops, topologically associated domains, and other higher-order structures. These 
          structures can have large impacts on gene expression regulation and evolution. For example, genes located in the same 
          domain may be co-regulated by sharing the same transcription factor molecules. Chromosome packing can also limit the 
          accessibility of genes in certain regions to transcription machineries and consequently control their activity.</p>
          
          <p>In this project, theoretical behaviors of popular models will be examined in conjunction with chromosome conformation 
          capture datasets from various organisms to gain further insight into the folding process of chromosomes. <!-- Parts of this 
          project will be performed in collaboration with <a href="http://iwasakilab.bs.s.u-tokyo.ac.jp/eindex.html">Dr. Wataru Iwasaki</a>
          ’s laboratory at the University of Tokyo and <a href="https://www.wistar.org/lab/ken-ichi-noma-phd">Dr. Ken-ichi Noma</a>’s 
          laboratory at the Wistar Institute.</p> -->
        </div>
      </div>
	  
	  <div class="row">
	    <div class="col-md-12" align="left">
	      <hr />
	      <h3><strong>Project 4:</strong> Visualization of biomedical data using virtual reality</h3>
	    </div>
	  </div>
	  <div class="row">
	    <div class="col-md-12">
	      <p>Even though we can perceive three dimensions, our visualization of data has largely been limited to 2-dimension media such as monitor screen or a 
	      piece of paper. With the emergence of affordable virtual reality technology, we can now explore the possibility of representing structural and 
	      non-structural biomedical data in 3-dimensional space and subsequent visualization in virtual reality. A primary goal of this project is to develop 
	      novel visualization methods and tools that facilitate comprehensive integration, visualization, and interpretation of complex biomedical datasets.</p>
        </div>
      </div>
	  
	</div>
	
  </body>
</html>
