<!--<div class="research-banner-bg">
  <div class="container">
     <div class="row">
       <div class="col-md-12" align="right">
         <h3><font color="grey">Expanding the horizon of computational biology research</font></h3>
       </div>
     </div>
  </div>
</div>-->
<!-- RESEARCH -->
<div class="container-fluid">
    <div class="row">
        <div class="col-md-12" align="left">
            <!-- <h2>Research</h2> -->
            <h3>
                The Computational Molecular Biology group is pursuing the following research topics as well as additional collaborative projects.
            </h3>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 1:</strong> Evolution of hybrid eukaryotic genomes</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Merging of distinct genomes into a new organism induces large-scale evolutionary processes that eventually lead to the emergence of species with novel phenotypes and capabilities -- many of which are important in agriculture and biotechnology.<p />
                </dd>
                <dd>
                    Understanding of the mechanisms that drive post-hybridization genome evolution can lead to improvement in the engineering of new hybrid species (such as yeasts) that possess desirable properties.<p />
                </dd>
                <dt>Our approaches</dt>
                <dd>
                    Together with our collaborators in Japan, we recently discovered a group of closely related, recently formed yeast hybrids that serves as a perfect platform for investigating the early stages of post-hybridization evolution.<p />
                </dd>
                <dd>
                    We are employing an integrated analysis of genome sequence, gene expression level, and chromatin structure data.
                </dd>
            </dl>
        </div><!---->
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/polyploid_project.png" alt="Project image" width="300" height="300">
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE LEFT -->
    <div class="row">
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/msms_project.png" alt="Project image" width="300">
        </div>
        <div class="col-md-8">
            <h3><strong>Project 2:</strong> Mass spectrometry-based <i>de novo</i> peptide sequencing</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Mass spectrometry technology enables high-throughput identification and quantification of peptides and proteins in a complex sample such as patient's blood and snake's venom.<p />
                </dd>
                <dd>
                    However, analysis of mass spectrometry data typically requires an input database containing known peptides and proteins. In other words, we cannot identify what we did not expect.<p />
                </dd>
                <dd>
                    In many applications, we simply do not know what peptides and proteins would be found and/or the possibilities are too vast to be computationally feasible.<p />
                </dd>
                <dt>Our approaches</dt>
                <dd>
                    We have developed deep neural network models, using millions of mass spectra, for identifying peptides without relying on protein databases. And we are still continuing to improve our peptide <i>de novo</i> sequencing capability even further.<p />
                </dd>
                <dd>
                    In addition to method development, we are also deploying our models to identify new peptides in various biological samples and disease models.<p />
                </dd>
            </dl>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 3:</strong> The distribution and community structure of microorganisms</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Microbes possess exceptional dispersability and population sizes that allow them to colonize new habitats.<p />
                </dd>
                <dd>
                    Study of global microbe-environment association pattern would provide insights into the forces and constraints that shape the distribution and community structure of these organisms.<p />
                </dd>
                <dt>Our approaches</dt>
                <dd>
                    Our recent work suggests that generalist microbes are responsible for spreading microbial species and genes across diverse environments.<p />
                </dd>
                <dd>
                    Knowledge of how microbial species and genes propagate may lead to development of better control and monitoring of future outbreaks.<p />
                </dd>
            </dl>
        </div>
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/microbiome_project.png" alt="Project image" width="300" height="300">
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE LEFT -->
    <div class="row">
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/software_project.png" alt="Project image" width="300" height="300">
        </div>
        <div class="col-md-8">
            <h3><strong>Project 4:</strong> Application of machine learning for biomedical research</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Machine learning has become a key approach in modern research and medical service.<p />
                </dd>
                <dd>
                    Research groups at Chulalongkorn University and Hospital are continually generating large amounts of data that require systematic managements and analyses.<p />
                </dd>
                <dt>Our approaches</dt>
                <dd>
                    We work closely with both physicians and computer engineers in order to fully understand the research goals (on the medical side) and the pros and cons of analysis techniques (on the engineering side).<p />
                </dd>
                <dd>
                    Python and MATLAB are primarily used.
                </dd>
            </dl>
        </div>
    </div>
</div>
