<!--<div class="research-banner-bg">
  <div class="container">
     <div class="row">
       <div class="col-md-12" align="right">
         <h3><font color="grey">Expanding the horizon of computational biology research</font></h3>
       </div>
     </div>
  </div>
</div>-->

<!-- RESEARCH -->
<div class="container-fluid">
    <div class="row">
        <div class="col-md-12" align="left">
            <!-- <h2>Research</h2> -->
            <h3>
                The Computational Molecular Biology group is pursuing 5 primary research topics as well as additional collaborative projects.
            </h3>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 1:</strong> Evolution of eukaryotic genomes following polyploidization events</h3>
            <dl>
                <dt>Background</dt>
                <dd>
                    Large-scale expansion of genome through polyploidization greatly increases gene repertoire and facilitates emergence of species with novel capabilities,
                    many of which are used in agriculture and biotechnology.<p />
                </dd>
                <dd>
                    Before reaping the long-term benefits, these polyploids must first cope with the sudden changes in genome.<p />
                </dd>
                <dt>Our approaches</dt>
                <dd>
                    Many evolutionary mechanisms during the early stages following polyploidization should prioritize genome restabilization.<p />
                </dd>
                <dd>
                    In collaboration with the University of Tokyo and the Japan Collection of Microorganisms, closely related fungal species that recently underwent genome
                    hybridization were discovered.<p />
                </dd>
                <dd>
                    Comprehensive analyses of the genome sequences, transcriptome profiles, and chromatin organizations of these species would shed light into the evolutionary
                    processes during the early stages of polyploidization.<p />
                </dd>
                <dd>
                    We are also interested in the expansion of gene families and signaling pathway components in model plants.
                </dd>
            </dl>
        </div><!---->
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/polyploid_project.png" alt="Project image" width="300" height="300">
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE LEFT -->
    <div class="row">
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/chromatin_project.png" alt="Project image" width="300" height="300">
        </div>
        <div class="col-md-8">
            <h3><strong>Project 2:</strong> Formation and organization of chromosome 3D structured</h3>
            <dl>
                <dt>Background</dt>
                <dd>
                    Genomes are located on chromosomes that fold in 3-dimensional space and form loops, topologically associated domains, and higher-order structures.<p />
                </dd>
                <dd>
                    These structures can impact gene expression regulation and evolution, for example, by enabling transcription factor sharing acriss genes located in the same
                    topologically associated domain.<p />
                </dd>
                <dd>
                    Chromosome packing can also limit the accessibility of genes in certain regions from transcription machineries.<p />
                </dd>

                <dt>Our approaches</dt>
                <dd>
                    Recent models of chromatin folding have been shown to capture many biologically important characteristics of chromosome 3D organization.<p />
                </dd>
                <dd>
                    We will explore theoretical behaviors of these models through mathematical models and <i>in silico</i> simulations.<p />
                </dd>
                <dd>
                    Chromosome conformation capture datasets from various organisms will be used to confirm theoretical results and to gain further insight into the folding
                    process of chromatin.
                </dd>
            </dl>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 3:</strong> Forces governing the distribution and community structure of microorganisms</h3>
            <dl>
                <dt>Background</dt>
                <dd>
                    Microbes possess exceptional dispersability and population sizes that allow them to invade new habitats.<p />
                </dd>
                <dd>
                    In response to such constant movements and competitions, microbes adopt two general strategies for survival: becoming generalists with larger gene
                    repertoires that let them cope with diverse environments, or becoming specialists with smaller genomes that are optimized for specific environments.<p />
                </dd>

                <dt>Our approaches</dt>
                <dd>
                    We recently proposed that generalists are responsible for spreading microbial species and genes as they expand across environments.<p />
                </dd>
                <dd>
                    Further validations of our model, for example, in the context of the spread of antibiotic resistance genes or in the context of disease outbreaks would
                    lead to practical medical applications.<p />
                </dd>
                <dd>
                    Other broad understandings of microbial community structures and dispersions may be derived from large-sacle publicly available microbiome datasets.
                </dd>
            </dl>
        </div>
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/microbiome_project.png" alt="Project image" width="300" height="300">
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE LEFT -->
    <div class="row">
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/software_project.png" alt="Project image" width="300" height="300">
        </div>
        <div class="col-md-8">
            <h3><strong>Project 4:</strong> Software development and application of machine learning techniques for large-scale biological datasets</h3>
            <dl>
                <dt>Background</dt>
                <dd>
                    Effective data analysis and visualization tools are essential in interpreting large-scale experimental datasets.<p />
                </dd>
                <dd>
                    Via collaborations with experimental research groups at Chulalongkorn University and other institutes, we have access to a large number of biomedical
                    datasets that need combined expertise in computational and biological sciences to analyze.<p />
                </dd>

                <dt>Our approaches</dt>
                <dd>
                    We currently supervise undergraduate Computer Engineering students at Chulalongkorn University to analyze large datasets and develop software tools that
                    facilitate such processes.<p />
                </dd>
                <dd>
                    All techniques, from statistical and mathematical modeling to advanced machine learning, are utilized.<p />
                </dd>
                <dd>
                    Heavy computations were carried out on our collaborators' high-performance CPU and GPU clusters.<p />
                </dd>
                <dd>
                    Python, Java, or MATLAB are typically used.
                </dd>
            </dl>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 5:</strong> Mining of scientific literatures</h3>
            <dl>
                <dt>Background</dt>
                <dd>
                    To conduct effective research, one must rigorously study the literatures and choose the appropriate experimental protocols and data analysis tools.<p />
                </dd>
                <dd>
                    New research reports and academic software get published at ever-increasing rates that can overwhelm any researcher.<p />
                </dd>
                <dd>
                    For example, given the sheer number of <a href="https://en.wikipedia.org/wiki/List_of_RNA-Seq_bioinformatics_tools" target="_blank">software tools</a>
                    for processing RNA-seq data, it is time-consuming to evaluate and select the best tools.<p />
                </dd>

                <dt>Our approaches</dt>
                <dd>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/" target="_blank">Pubmed</a> maintains a database of millions of published biomedical literatures.<p />
                </dd>
                <dd>
                    Mining of these contents can reveal patterns behind major trends such as research topic popularity or spread of diseases.<p />
                </dd>
                <dd>
                    We can also automatically and systematically compare software tools that perform similar tasks based on the profiles of their authors, citation trends,
                    and other supporting information.
                </dd>
            </dl>
        </div>
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/white_placeholder_300x300px.png" alt="Project image" width="300" height="300">
        </div>
    </div>
</div>