<!--<div class="research-banner-bg">
  <div class="container">
     <div class="row">
       <div class="col-md-12" align="right">
         <h3><font color="grey">Expanding the horizon of computational biology research</font></h3>
       </div>
     </div>
  </div>
</div>-->

<!-- RESEARCH -->
<div class="container-fluid">
    <div class="row">
        <div class="col-md-12" align="left">
            <!-- <h2>Research</h2> -->
            <h3>
                The Computational Molecular Biology group is pursuing the following research topics as well as additional collaborative projects.
            </h3>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 1:</strong> Evolution of eukaryotic genomes following polyploidization events</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Large-scale expansion of genome through polyploidization greatly increases gene repertoire and facilitates emergence of species with novel capabilities,
                    many of which are used in agriculture and biotechnology.<p /></dd>
                <dt>Our approaches</dt>
                <dd>
                    We recently discovered a group of closely related yeast hybrids and are now analyzing their genome sequences, transcriptome profiles, and chromatin organizations
                    in order to identify key post-polyploidization evolutionary processes.<p /></dd>
                <dd>
                    We are also interested in how polyploidization direct the expansion of gene families and signaling pathway components in plants.</dd>
            </dl>
        </div><!---->
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/polyploid_project.png" alt="Project image" width="300" height="300">
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE LEFT -->
    <div class="row">
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/chromatin_project.png" alt="Project image" width="300" height="300">
        </div>
        <div class="col-md-8">
            <h3><strong>Project 2:</strong> Organization of chromosome 3D structure</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Genomes are located on chromosomes that fold in 3-dimensional space and form loops, topologically associated domains, and higher-order 
                    structures that can impact gene expression regulation and evolution.<p /></dd>
                <dt>Our approaches</dt>
                <dd>
                    Recent models of chromatin folding have been shown to capture many biologically important characteristics of chromosome 3D organization.<p /></dd>
                <dd>
                    We will explore theoretical properties of these models through mathematical models and <i>in silico</i> simulations and validate our findings
                    using publicly available chromatin conformation capture data.<p /></dd>
            </dl>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 3:</strong> The distribution and community structure of microorganisms</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Microbes possess exceptional dispersability and population sizes that allow them to colonize new habitats.
                    Study of global microbe-environment association pattern should provide insights into the forces and constraints 
                    that govern the dynamics of microbial distribution and community structure.<p /></dd>
                <dt>Our approaches</dt>
                <dd>
                    Our recent work suggests that generalist microbes are responsible for spreading microbial species and genes across diverse environments.
                    Further validations of our model, for example, in the context of the spread of antibiotic resistance genes or in the context of disease outbreaks would
                    lead to practical medical applications.<p /></dd>
                <dd>
                    We will mine large-scale microbiome datasets to learn other broad understandings of microbial community structures and dispersions.
                </dd>
            </dl>
        </div>
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/microbiome_project.png" alt="Project image" width="300" height="300">
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE LEFT -->
    <div class="row">
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/software_project.png" alt="Project image" width="300" height="300">
        </div>
        <div class="col-md-8">
            <h3><strong>Project 4:</strong> Software development and application of machine learning for biomedical research</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    Effective data analysis and visualization tools are essential for interpreting any large-scale experimental dataset.<p /></dd>
                <dd>
                    Many research groups at Chulalongkorn University as well as our collaborators generate large amount of data
                    that need combined expertise in computational and biological sciences to analyze.<p /></dd>
                <dt>Our approaches</dt>
                <dd>
                    We utilize statistical and machine learning techniques, with access to high performance CPU and GPU clusters.<p /></dd>
                <dd>
                    Python and MATLAB are primarily used, followed by Java and C++ knowledge</dd>
            </dl>
        </div>
    </div>

    <hr />
    <!-- IMAGE ON THE RIGHT -->
    <div class="row">
        <div class="col-md-8">
            <h3><strong>Project 5:</strong> Mining of scientific literatures</h3>
            <dl>
                <dt>Motivation</dt>
                <dd>
                    To conduct effective research, one must rigorously study the literatures and choose the appropriate experimental protocols and data analysis tools. 
                    However, new research reports and academic software get published at ever-increasing rates.<p /></dd>
                <dd>
                    For example, given the sheer number of <a href="https://en.wikipedia.org/wiki/List_of_RNA-Seq_bioinformatics_tools" target="_blank">software tools</a>
                    for processing RNA-seq data, it is time-consuming to evaluate and select the best tools.<p /></dd>

                <dt>Our approaches</dt>
                <dd>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/" target="_blank">Pubmed</a> maintains a database of millions of published biomedical literatures.<p /></dd>
                <dd>
                    We are interested in developing an automatic curation system that can capture and track current global research interests as well as
                    identify promising bioinformatics tools.
                </dd>
            </dl>
        </div>
        <div class="col-md-4" align="center">
            <img class="rounded" src="content/white_placeholder_300x300px.png" alt="Project image" width="300" height="300">
        </div>
    </div>
</div>